CI: nightly broken

The nightly ci is currently broken:

```
Created /home/runner/work/plugins/plugins/lightning/contrib/msggen/msggen/schema.json from 2 files
cargo build --quiet --bin cln-grpc
make: cargo: No such file or directory
make: *** [plugins/Makefile:300: target/debug/cln-grpc] Error 127
Error: Process completed with exit code 2.
```

I've tried to debug this but couldn't find the problem.

It works locally and in the cln repo. 

I believe it has something to do with the change in the plugins/Makefile adding MSGGEN_GEN_ALL:

https://github.com/ElementsProject/lightning/commit/374db239bec889356996a3fb37d57d1cc8141e49#diff-7eba0837a02c2deea05123583489cc7941f60b687718194ad0b30077c3181c62R297

Definition:
```
MSGGEN_GEN_ALL = \
	contrib/msggen/msggen/schema.json
```

Not to be confused with MSGGEN_GENALL

--- a/.github/workflows/nightly.yml
+++ b/.github/workflows/nightly.yml
@@ -79,8 +79,8 @@ jobs:
         pip install --user contrib/pyln-client contrib/pyln-testing flaky
 
         ./configure --disable-valgrind
-        make -j 16
-        sudo make install
+        make -j $(nproc)
+        sudo -E bash -c "source $HOME/.cargo/env && rustup default stable && make install"
 
     - name: Test with pytest
       run: |

